tools/distpartitioning/utils.py

```diff
@@ -420,3 +422,26 @@ def get_idranges(names, counts):
 
     return tid_dict, gid_dict
 
+
+def mem_snapshot(tag, rank):
```

Comment:
It is not a well-known convention to use mem for memory. Change it to memory_snapshot?

e.g. https://pytorch.org/docs/stable/generated/torch.cuda.memory_snapshot.html

Reply:
Done as suggested.